### PR TITLE
Update openstackgit playbook to synchronise to rpc-repo

### PR DESCRIPTION
**Pull request by odyssey4me**
This patch updates the git sync playbook to synchronise to rpc-repo
when it's complete.

### CODE REVIEW

File: openstackgit-update.yml [hunk 1]
```diff
@@ -58,7 +49,31 @@
       with_items: "{{ local_packages.results.0.item.remote_package_parts }}"
       register: git_clone
       until: git_clone | success
-      retries: 2
+      retries: 5
+      delay: 5
+
+  vars:
```
Comment (evrardjp) [inline openstackgit-update.yml]: For readability, I'd put these vars just above tasks:

File: openstackgit-update.yml [hunk 2]
```diff
@@ -23,7 +23,7 @@
 # is expected to contain more repositories than RPC will need, but
 # will always have a complete set.
 
-- name: Update the openstackgit clones
+- name: Fetch the latest openstackgit clones
   hosts: localhost
   connection: local
```
Comment (evrardjp) [inline openstackgit-update.yml]: you don't seem to include artifact version, which should be in artifacting-vars.yml

File: openstackgit-update.yml [hunk 3]
```diff
@@ -33,22 +33,13 @@
         path: "{{ working_dir }}/openstackgit"
         state: directory
```
Comment (evrardjp) [inline openstackgit-update.yml]: why did you remove the git clone? Is this playbook supposed to be included from another playbook?

File: openstackgit-update.yml [hunk 4]
```diff
@@ -58,7 +49,31 @@
...
+
+    - name: Ensure the openstackgit folder is present
+      file:
+        path: "/var/www/repo/openstackgit"
+        state: directory
+
+    - name: Synchronize cache
+      synchronize:
+        src: "{{ working_dir }}/openstackgit"
+        dest: "/var/www/repo/openstackgit"
```
Comment (evrardjp) [inline openstackgit-update.yml]: you don't use for dest:  "| dirname" ?
You'll have openstackgit/openstackgit I think.

File: openstackgit-update.yml (hunk 2 above)
Comment (odyssey4me) [inline openstackgit-update.yml]: The git repositories do not need an artifact version. They simply need the latest copy of the git repo. An up to date copy will always contain the SHA's required for previous series and all versions.

File: openstackgit-update.yml (hunk 3 above)
Comment (odyssey4me) [inline openstackgit-update.yml]: The git clone of openstack-ansible is being done in the jenkins job itself. See https://github.com/rcbops/jenkins-rpc/pull/727

File: openstackgit-update.yml (hunk 4 above)
Comment (odyssey4me) [inline openstackgit-update.yml]: Not sure myself - I was just going to kinda see what happens and adjust afterwards.
